README.md

```diff
@@ -1,12 +1,72 @@
-In one terminal, run:
+# django-taskq
+
+[![MIT](https://img.shields.io/github/license/ipsosante/django-taskq.svg)](https://tldrlegal.com/license/mit-license)
+[![Build Status](https://img.shields.io/travis/ipsosante/django-taskq/master.svg)](https://travis-ci.org/ipsosante/django-taskq)
+[![Codecov](https://img.shields.io/codecov/c/github/ipsosante/django-taskq/master.svg)](https://codecov.io/gh/ipsosante/django-taskq)
+
+django-taskq is an asynchronous task queue/job queue for Django. It's a lightweight alternative to [Celery](http://www.celeryproject.org/).
+
```

Comment:
In case of an unhandled exception in your Django website, If you use the same database than taskq, jobs will be automatically cancelled with transaction rollback.